wandb/README.md

```diff
@@ -0,0 +1,22 @@
+# Tracking optimization process with Weights & Biases
+
+![wandbui](https://user-images.githubusercontent.com/37713008/133468378-3eae55e3-dd07-4e87-b3cf-f60c06fd061d.png)
+
+Optuna example that optimizes a classifier configuration for the Iris dataset usinng
```

Comment:
```suggestion
Optuna example that optimizes a classifier configuration for the Iris dataset using
```

Reply:
Fixed in 03805bd1f70073201ac4e53bbeeff82820d26e69.